package.json

```diff
@@ -45,10 +45,12 @@
   },
   "dependencies": {
     "@shiftcommerce/shift-node-api": "^0.4.1",
-    "@shiftcommerce/shift-react-components": "^0.9.1",
+    "@shiftcommerce/shift-react-components": "^0.10.0",
     "axios": "^0.18.0",
     "js-cookie": "^2.2.0",
     "qs": "^6.6.0",
+    "react-redux": "^5.0.7",
+    "redux": "^3.7.2",
```

Comment:
These are added here because they were moved to `devDependencies` in shift-react-components.